src/styles/style.css

```diff
@@ -5,22 +5,34 @@
 }
 
 #emote-menu-button {
-	background-image: url(../images/smiley.png) !important;
-	background-position: 50%;
+	background-image: url(../images/smiley.svg) !important;
+	background-size: 20px;
+	background-position: center;
 	background-repeat: no-repeat;
 	cursor: pointer;
 	height: 30px;
 	width: 30px;
+	-webkit-filter: grayscale(100%);
```

Comment:
Just for reference: any prefixed properties (`-webkit-*`) are not needed as [autoprefixer](https://github.com/cletusc/Userscript--Twitch-Chat-Emotes/blob/cb1f75594480e65a675b9cd9186607ccb0ca71ff/gulp/tasks/styles.js#L4) is used.